images/build/python/prereqs/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Install Python build prerequisites
-FROM buildpack-deps:stable
+FROM buildpack-deps:stretch
```

Comment:
Should we even update the run-base image too? From `buildpack-deps:curl` to `buildpack-deps:stretch-curl`? cc @gpcastro

Reply:
good point - done